framework/src/modules/chain/synchronizer/block_synchronization_mechanism.js

```diff
@@ -78,6 +79,14 @@ class BlockSynchronizationMechanism extends BaseSynchronizer {
 					block: receivedBlock,
 				});
 			}
+
+			if (error instanceof AbortError) {
+				return this.logger.info(
+					{ error, reason: error.reason },
+					`Aborting synchronization mechanism with reason: ${error.reason}`,
```

Comment:
Since we already have reason as part of json we do not need it again, maybe change it to `Aborting synchronization mechanism!!`?

Reply:
Done :)